src/Message.vue

```diff
@@ -67,6 +77,7 @@
 <script>
 import TextMessage from './messages/TextMessage.vue'
 import FileMessage from './messages/FileMessage.vue'
+import ImageMessage from './messages/ImageMessage.vue'
```

Comment:
This was never registered in the `components` section below on line 87, and thus results in the error:

![image](https://user-images.githubusercontent.com/26761352/101955348-a2560380-3c23-11eb-8e23-da11f0a86d58.png)